src/main/kotlin/no/nav/syfo/api/endpoints/VedtakEndpoints.kt

```diff
@@ -22,6 +23,7 @@ import java.util.UUID
 const val vedtakUUIDParam = "vedtakUUID"
 const val apiBasePath = "/api/internad/v1/frisktilarbeid"
 const val vedtakPath = "/vedtak"
+const val validerPath = "/valider"
```

Comment:
Burde det vært navngitt noe i retning vilkår? 🤔 `/vedtak-vilkar` / `/vilkar-for-vedtak` for eks. Tanker om det? 😊

Reply:
Gjerne det! Fikser det og legger på noe testkode.